apps/image-focal-point/src/index.jsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ReactDOM from 'react-dom';
-import { Note } from '@contentful/forma-36-react-components';
+import { Note } from '@contentful/f36-components';
```

Comment:
I think one of the updates with `note` was the switch to use a generic `variant` prop instead of the component specific `noteType` prop. I didn't see that change in here and I'm figuring that's because this app isn't in typescript and so it didn't yell about it? But I wonder if the note's variant is actually being respected since we're passing in a deprecated prop.

Reply:
good to know! will fix